modules/ROOT/pages/create-application.adoc

```diff
@@ -42,7 +42,7 @@ image:images/getting-started-tutorial/create-application/create-application.gif[
 The look & feel can be fully customized by modifying the xref:layouts.adoc[layout] and adding a custom xref:customize-living-application-theme.adoc[theme].
 ====
 
-The application is ready to be used. Now you need to deploy it in Bonita Portal:
+The application is ready to be used. Now you need to deploy it in Bonita Applications:
```

Comment:
The application is ready to be used. Now you need to deploy it in Bonita Applications: => The application is ready to be deploied in Bonita Applications. Once done, your application is ready to be used. 

from user perspective "ready to be used" means no other activities are to be done, but we still have the deploiement to do, so, for me the application is not ready to be used. Maybe ready to be consumed, but the use is for final users.

Reply:
Added